framework/src/org/apache/cordova/CordovaPlugin.java

```diff
@@ -40,6 +42,7 @@ Licensed to the Apache Software Foundation (ASF) under one
     public CordovaInterface cordova;
     protected CordovaPreferences preferences;
     private String serviceName;
+    protected String [] permissions;
```

Comment:
I feel this is somewhat confusing. What is the derived plugin supposed to with this field? I do not see any of the core plugins doing anything with this field. Did I miss something?